README.md

```diff
@@ -8,3 +8,56 @@ This repo has 3 main parts:
 2) ```./ui``` The mcp ui. This is an extension to the OpenShift UI to give a mobile centric view and set of features.
 
 3) ```./docs``` This holds design docs, architectural docs, use cases and development guides etc.
+
+## Prerequisites
+
+* `oc` cli >= 3.6.0-rc0 https://github.com/openshift/origin/releases (Known issue with service-catalog in 3.6.0)
```

Comment:
Since we're using the `--version` flag is this still applicable? I'm using `oc v3.6.0+c4dd4cf` and everything ran fine.